src/test/Makefile

```diff
@@ -265,15 +269,24 @@ telemetry_ibus_unittest_SRC := \
 		$(USER_DIR)/telemetry/ibus_shared.c \
 		$(USER_DIR)/telemetry/ibus.c
 
+timer_definition_unittest_EXPAND := yes
 
-transponder_ir_unittest_SRC := \
-	        $(USER_DIR)/drivers/transponder_ir_ilap.c \
-	        $(USER_DIR)/drivers/transponder_ir_arcitimer.c
+# SITL is a simulator; NERO and STM32F7X2 don't define USED_TIMERS in target.h.
+timer_definition_unittest_BLACKLIST := STM32F7X2 SITL NERO
 
+timer_definition_unittest_SRC = \
+		$(TARGET_DIR)/$(call get_base_target,$1)/target.c
 
-type_conversion_unittest_SRC := \
```

Comment:
Where did this guy go?

Reply:
I have no clue. :grin: My new consistency checks complained that there was no `type_conversion_unittest.cc`, so I just removed the variable.